.github/renovate.json5

```
@@ -416,6 +416,7 @@
         '!/credativ/.+/',
         '!/envoyproxy/.+/',
         '!/fluent/.+/',
+        '!/persesdev/.+/',
```

```suggestion
        '!/perses/.+/',
```

The github-releases manager is using the `sourceRepository` field ([ref](https://github.com/gardener/ci-infra/blob/a6c05dce7bf59afe3b4996c29b4b210f2b301ab6/config/renovate/imagevector.json5#L16)).
The github organization is `perses` ([ref](https://github.com/gardener/gardener/blob/213ad5f004aeb14c7e31cacb32b81efbfa56321d/imagevector/containers.yaml#L388)), so we should use this term for the matcher.

---

Of course, thanks for catching that!